components/event-sources/README.md

```diff
@@ -0,0 +1,55 @@
+# Knative event sources for Kyma
+
+Available event sources:
+
+* `MQTTSource`: for SAP S/4 applications emitting MQTT events.
```

Comment:
sap only?

Reply:
Yes, as discussed this is a highly-specific duct tape. Not sure why this has to be part of the open-source offering, but it is how it is.